content/projects/coursera/it-automation/course-1-python-crash-course/_index.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
this should be in content/coursera/... we are in the process of refactoring and wont be using the /projects folder any more